pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -349,6 +362,7 @@ func (ic *ifConfigurator) configureContainerLink(
 	podSriovVFDeviceID string,
 	result *current.Result,
 	containerAccess *containerAccessArbitrator,
+	mac string,
```

Comment:
can this be applicable for the SR-IOV case as well? maybe by calling `netlink.LinkSetVfHardwareAddr`?
Don't know if @jianjuns or @tnqn are aware of any limitation (e.g. only works for the baremetal case)?

Reply:
According to https://github.com/k8snetworkplumbingwg/sriov-cni/blob/3271b0f594b6d814f2e83a43589d99c6bd1882f3/pkg/utils/utils.go#L301-L315, it works for VM too, but the setting is asynchronously realized for some drivers.

Reply:
In this PR i am thinking to handle VLAN network case, is it ok if i can handle SR-IOV case in separate PR?

Reply:
@KMAnju-2021 ok, please open an issue to track this

@tnqn thanks for the link